### PR TITLE
Move inline style declarations to stylesheet

### PR DESCRIPTION
**Pull request by maddyrandle**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling (Styling to new or existing files)
- [ ] Testing
### Detailed Description
* Style PlantInfo page MVP
* Set up navigation that allows a user to click a button in the PlantInfo page that navigates them back to search 
### Why is this change required? What problem does it solve?
* We needed to style the app in a way that looks attractive, and also is rendering our page content in an organized way
* The navigation functionality allows a user to leave the PlantInfo page after viewing a selected plant's info
### Was any previously written code changed?
- Moved inline styling declarations out of `/PlantInfo.js` and into the stylesheet `styles/styles.js`
- I deconstructed 'navigation' from the props being passed to PlantInfo.js

### Directory modified:
- `components/pages/PlantInfo.js`
- `styles/styles.js`
- `components/pages/Search.js`
- `components/PlantItem.js`

### CODE REVIEW

File: components/PlantItem.js [hunk 1]
```diff
@@ -3,18 +3,20 @@ import { View, Text, Image, TouchableHighlight} from 'react-native';
...
-export default function PlantItem ({ title, image, searchNavigation }) {
-  return(
+export default function PlantItem ({ title, image, searchNavigation, tabNavigation: {  dangerouslyGetParent} }) {
+  return (
     <TouchableHighlight
-      onPress={() => searchNavigation.navigate('PlantInfoPage')}
-    >
+      onPress={() => {
+        searchNavigation.navigate('PlantInfoPage');
+        dangerouslyGetParent().setOptions({ tabBarVisible: false });
```
Comment (cmpprg) [inline components/PlantItem.js:11]: For Reference: Maybe there is a way that we can set this in the tabNavigator instead of toggling back and forth?

**Review by cmpprg — APPROVED** [review]
This looks really good, Thanks for all the hard work!!

File: components/pages/MyGarden.js [hunk 2]
```diff
@@ -3,13 +3,15 @@ import { View, Text, Button } from 'react-native';
 
 import styles from '../../styles/styles';
 
-export default function MyGarden(props) {
+export default function MyGarden({ navigation }) {
```
Comment (kimalajoy) [inline components/pages/MyGarden.js:6]: Love me some destructuring

**Review by kimalajoy — COMMENTED** [review]
Looks great!

**Pull request merged**